receiver/example_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package receiver_test
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.opentelemetry.io/collector/receiver"
+)
+
+var typeStr = component.MustNewType("examplereceiver")
```

Comment:
The type doesn't usually have the [component.Kind](https://pkg.go.dev/go.opentelemetry.io/collector/component#pkg-variables)
```suggestion
var typeStr = component.MustNewType("example")
```